### ISSUE TITLE
Repo-updater 'perms_syncer_sync_errors'

### ISSUE THREAD

**Issue by daxmc99** [sourcegraph/sourcegraph#14884]
- **Sourcegraph version:** 
Sourcegraph.com 

This alert has fired multiple times in the past 24-hrs https://sourcegraph.slack.com/archives/C0J618TTM/p1603162583038100 

There seem to be two repeating logs

`"pq: temporary file size exceeds temp_file_limit (1025563kB)"` & `fetch user permissions: ... 401 Bad Credentials` 

[link](https://console.cloud.google.com/logs/query;query=resource.type%3D%22k8s_container%22%0Aresource.labels.cluster_name%3D%22cloud%22%0Aresource.labels.namespace_name%3D%22prod%22%0Aresource.labels.container_name%3D%22repo-updater%22%0Aresource.labels.location%3D%22us-central1-f%22;timeRange=PT1H?q=search&referrer=search&project=sourcegraph-dev&query=%0A)

**Comment by daxmc99** [comment]
Possibly fixed by https://github.com/sourcegraph/sourcegraph/pull/14793

**Comment by unknwon** [comment]
I've filed https://github.com/sourcegraph/sourcegraph/issues/14899 to track the problem on permissions syncing side. Before that is done, should we silence the alert?

**Comment by unknwon** [comment]
@bobheadxi I'm not sure but did you add [this external service](https://sourcegraph.com/site-admin/external-services/RXh0ZXJuYWxTZXJ2aWNlOjIx) that syncs repos from https://github.com/ubclaunchpad? Right now, the external service config contains `authorization` field which effectively turns on repository permissions syncing on Sourcegraph Cloud (which is not a problem by itself).

Per #14899, it suggests we're not ready for that. So my question is that does https://github.com/ubclaunchpad contains private repos? If not, I would like remove the `authorization` field from this external service config.

Also cc @ryanslade and @asdine to confirm we would skip private repos on listing user-added external services as of now?

**Comment by ryanslade** [comment]
@unknwon Yes, for user added external services we filter out all private repos:

https://github.com/sourcegraph/sourcegraph/blob/main/cmd/repo-updater/repos/syncer.go#L239-L242

**Comment by unknwon** [comment]
Thanks @ryanslade! @bobheadxi please confirm you can't search any private repos from https://github.com/ubclaunchpad on Sourcegraph Cloud. If that is true, I'll remove the `authorization` field until #14899 is fixed.

**Comment by tsenart** [comment]
@ryanslade, @unknwon: How did  a non Sourcegrapher add external services in sourcegraph.com?

**Comment by unknwon** [comment]
@tsenart Robert is a sourcegrapher 😅

**Comment by tsenart** [comment]
@unknwon: Sorry, didn't realise it was @bobheadxi :D

**Comment by bobheadxi** [comment]
Yes that's me, and thats my external service config! I just confirmed that [one of our private repos is not present](https://sourcegraph.com/search?q=repo:ubclaunchpad/leads&patternType=literal), which I noticed earlier but assumed it just wasn't supported - didn't realize it would be the cause of alerts!

I've removed the `authorization` block - just added it out of curiosity because it showed up in autocomplete 😅 Let me know if there's anything else I should change!
